### PR TITLE
explaintest: use mysql-tester in explaintest

### PR DESCRIPTION
**Pull request by Defined2014**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/45961

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: cmd/explaintest/s.zip
Comment (hawkingrei) [inline cmd/explaintest/s.zip:1]: What is it ?

Comment (Defined2014) [reply cmd/explaintest/s.zip:1]: some stats information, I don't know why it's a zip file.

File: cmd/explaintest/run-tests.sh [hunk 1]
```diff
@@ -17,16 +17,13 @@ TIDB_TEST_STORE_NAME=$TIDB_TEST_STORE_NAME
 TIKV_PATH=$TIKV_PATH
```
Comment (hawkingrei) [inline cmd/explaintest/run-tests.sh:17]: Bazel's ```sh_test``` can be used here.

Comment (Defined2014) [reply cmd/explaintest/run-tests.sh:17]: OK, I will do it later with bazel support.

**Review by xhebox — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Review by hawkingrei — APPROVED** [review]
I hope to see Bazel Cache available here as soon as possible.

**Pull request merged**